pyproject.toml

```diff
@@ -1,13 +1,13 @@
 [tool.poetry]
 name = "datasetinsights"
-version = "0.2.0"
+version = "1.0.0"
```

Comment:
Any reason why we want to upgrade the version?

Reply:
Usually this version does not matter. The pypi package version is based on the release tag which overrides this version flag. I updated it to reduce confusion because we recently released 1.0.